cli/dcoscli/node/main.py

```diff
@@ -23,7 +23,6 @@
     --version               Show version
 """
 
-import os
```

Comment:
This file is for `dcos node`, not `dcos service`.  `dcos service` doesn't have any `--option` param.  Only `--ssh-config-file`